index.html

```diff
@@ -341,13 +341,16 @@ <h2>Contact me</h2>
         <input name="Name" type="text" placeholder="Input your name" maxlength="30" required />
         <input name="Email" type="email" placeholder="Input your email" required />
         <textarea name="message" id="message" placeholder="Write your massege here" maxlength="500" required></textarea>
-        <small></small>
+        <span></span>
+
         <button type="submit">Get in touch</button>
       </form>
     </section>
   </div>
   <script src="./javascript/main script.js"></script>
   <script src="./javascript/popup.js"></script>
+  <script src="./javascript/formvalidation.js"></script>
```

Comment:
- [ ] Good work with the validation, but unfortunately, it's not working because you attached it to your HTML file with the path `./javascript` while your folder is named `JavaScript`. Kindly fix the path and check if everything is working alright. 🤝️

Reply:
-  Well resolved @MishDM 👍🏽 
<img width="716" alt="Screenshot 2023-02-02 at 16 52 40" src="https://user-images.githubusercontent.com/58692590/216374132-8999cc91-4bfd-4635-9294-8af10d360e1d.png">